core/server/apps/amp/lib/views/amp.hbs

```diff
@@ -42,9 +42,7 @@
             <header class="post-header">
                 <h1 class="post-title">{{title}}</h1>
                 <section class="post-meta">
-                    {{#author}}
-                    <p class="author">by <a href="{{url}}">{{name}}</a></p>
-                    {{/author}}
+                    <p class="author">by {{authors}}</p>
```

Comment:
@peterzimon Is that okay? Wasn't sure because of https://github.com/TryGhost/Casper/blob/master/partials/post-card.hbs#L22. IMO There is no need for making this pretty or adding more effort into this, because it's the AMP feature. But wanted to double check with you, because it's a design change :)

Reply:
I think this is fine. the output will be practically the same for one author and a list of authors linked to author listing for multiple.